### DISCUSSION TITLE
v0.2.3

### DISCUSSION THREAD

**Discussion by gaocegege** [tensorchord/envd#913] [Version releases]
## What's Changed
* docs: updated CHANGELOG.md by @github-actions in https://github.com/tensorchord/envd/pull/907
* docs: updated CHANGELOG.md by @github-actions in https://github.com/tensorchord/envd/pull/911
* fix: Use macos 11 by @gaocegege in https://github.com/tensorchord/envd/pull/912

**Full Changelog**: https://github.com/tensorchord/envd/compare/v0.2.2...v0.2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/tensorchord/envd/releases/tag/v0.2.3'>v0.2.3</a>.</em>

**Comment by Xuanwo** [comment]
Hugo made a change to its release naming's recently to adopt the same style with `GOARCH` and `GOOS`.

> We have standardised the archive names for the release archives (as you can see further below). Hugo has since the first version used a rather odd and non-standard mixed case naming of the archive files (e.g. hugo_0.102.3_OpenBSD-64bit.tar.gz). We now use the standard GOOS/GOARCH values as-is, which makes it easier for people to script against. To avoid breakage when running on Netlify and similar, we create aliases for the most commonly downloaded Linux-archives on the old format and will continue to do so in the foreseeable future.

Ref: https://github.com/gohugoio/hugo/releases/tag/v0.103.0

I think we can adopt this too.

**Reply by gaocegege** [reply]
SGTM!
